src/pages/api/fake_only/DEPRECATED_DIRECTORY.txt

```diff
@@ -0,0 +1 @@
+Use the "pages/api/_fake" directory!
```

Comment:
Do we really need this message? Who is looking here but us?

Reply:
yea maybe just deprecate fake_only ya? there's only one endpoint

Reply:
Hmm, so what are the different types of endpoints that the `fake` would have? So far I can think of three kinds that have come up

1. Direct access to internals (generic and work the same for every fake), e.g. we could have `_fake/info` return the `package.json' or such metadata. Also '_fake/database` as added here
2. Scenarios to setup or emulate behavior.
3. Endpoints to trigger internal state changes, like this complete connect webview.

Are 2 and 3 the same? Do we even need 2 or 3 if we have `_fake/rpc` where we can send db mutation commands?

Reply:
2 & 3 can be the same if we introduce rpc, which I'm beginning to think is a great hack

I would love to have some pages that literally print tables to explain what's going on in the fake, maybe one could even be auto-generated if not manually created based on the JSON (although reading JSON is nice, I think tables can sometimes improve relational items- although tbd on how we would do that automatically)

Reply:
haha makes me want sqlite as our backing engine... super tempting but ultimately would probably be controversial if we went to OS. Maybe as a future option